### ISSUE TITLE
CaiT-XXS24_224 pretrained model's performance is only about 67% in ImageNet?

### ISSUE THREAD

**Issue by JiaDingCN** [facebookresearch/deit#88]
Hi, thank you very much yesterday for providing CaiT-XXS24_224 pretrained model. But I just load it and run one epoch in ImageNet with 8 v100 cards and the Acc1 accuracy is only **67.07%**(In paper it's 77.6). I set all hyper-parameters as told in the paper except batch and lr(both x0.5) . So I wonder is it a correct version of pretrained model, or anything  I did is wrong? I will really appreciate it if you could help me figure it out.

My command(based on project [pytorch-image-models](https://github.com/rwightman/pytorch-image-models):
`python -m torch.distributed.launch --nproc_per_node=8 --nnodes=1 --node_rank=0 --master_addr=<addr> --master_port=<port> train.py /cache/data/imagenet/ --output /cache/models/ --init_method <method> --model cait_XXS24_224  -b 64  --epochs 300  --opt AdamW --weight-decay 0.05 --lr 5e-4 --warmup-epochs 5 --mixup 0.8 --cutmix 1.0 --reprob 0.25 --drop-path 0.1 --model-ema-decay 0.99996 --opt-eps 1e-8  --sched consine --warmup-lr 1e-6 --decay-rate 0.1 --aa rand-m9-mstd0.5-inc1 --train-interpolation bicubic --reprob 0.25 --remode pixel  --pretrained`(this command works well when training DeiT)

Log:
```
Model cait_XXS24_224 created, param count:11956264
Data processing configuration for current model + dataset:
	input_size: (3, 224, 224)
	interpolation: bicubic
	mean: (0.485, 0.456, 0.406)
	std: (0.229, 0.224, 0.225)
	crop_pct: 0.9
AMP not enabled. Training in float32.
Using native Torch DistributedDataParallel.
Scheduled epochs: 300
Train: 0 [   0/2502 (  0%)]  Loss:  3.295080 (3.2951)  Time: 3.171s,  161.45/s  (3.171s,  161.45/s)  LR: 5.000e-04  Data: 1.050 (1.050)
Train: 0 [  50/2502 (  2%)]  Loss:  3.835494 (3.5653)  Time: 0.408s, 1253.80/s  (0.485s, 1055.93/s)  LR: 5.000e-04  Data: 0.011 (0.031)
Train: 0 [ 100/2502 (  4%)]  Loss:  4.138270 (3.7563)  Time: 0.597s,  857.88/s  (0.450s, 1138.25/s)  LR: 5.000e-04  Data: 0.010 (0.021)
...
Train: 0 [2500/2502 (100%)]  Loss:  4.182121 (4.1164)  Time: 0.337s, 1518.49/s  (0.415s, 1233.23/s)  LR: 5.000e-04  Data: 0.010 (0.011)
Train: 0 [2501/2502 (100%)]  Loss:  4.102993 (4.1161)  Time: 0.374s, 1369.13/s  (0.415s, 1233.28/s)  LR: 5.000e-04  Data: 0.000 (0.011)
Test: [   0/97]  Time: 1.121 (1.121)  Loss:  1.0473 (1.0473)  Acc@1: 76.9531 (76.9531)  Acc@5: 92.9688 (92.9688)
Test: [  50/97]  Time: 0.100 (0.224)  Loss:  1.6498 (1.2922)  Acc@1: 60.3516 (70.4963)  Acc@5: 86.7188 (90.7629)
Test: [  97/97]  Time: 0.826 (0.217)  Loss:  1.2688 (1.4506)  Acc@1: 71.1310 (67.0660)  Acc@5: 88.6905 (88.3720)
Current checkpoints:
 ('/cache/models/train/20210415-104231-cait_XXS24_224-224/checkpoint-0.pth.tar', 67.06599999023437)
```

**Comment by TouvronHugo** [comment]
Hi @JiaDingCN ,
Thanks for your question.

I have just evaluated this model and I have the accuracy of the paper. 
Have you tried to evaluate it without doing 1 epoch of training to see if you had the correct accuracy? 
Do you use the file cait_models.py as is or do you adapt it to include it in your code?
I see that you use `crop_pct: 0.9` in our case we use `crop_pct: 1.0` but that shouldn't make much difference.

Best,

Hugo

**Comment by JiaDingCN** [comment]
Hi,thanks for your help again. 
I just tried to evaluate without 1 epoch training and the accuracy is right now(78.53%). It's interesting that accuracy change markedly between two epochs. Thank you again for your solid work and warmed help!
